spec/index.bs

```diff
@@ -1835,27 +1832,83 @@ dictionary GPURenderBundleEncoderDescriptor : GPUObjectDescriptorBase {
 Queues {#queues}
 ================
 
+<script type=idl>
+dictionary GPUTextureDataLayout {
+    GPUBufferSize offset = 0;
+    required unsigned long rowPitch;
+    required unsigned long imageHeight;
+};
```

Comment:
Nit: Please spec the units of these fields.  I presume that `offset`, `rowPitch` and `imageHeight` are in bytes?

Reply:
`offset` and `rowPitch` are in bytes but `imageHeight` is in texels. We discussed this a while ago in . https://github.com/gpuweb/gpuweb/issues/148